config/agents/svc/kustomization.yaml

```diff
@@ -51,4 +51,3 @@ images:
   newTag: latest
 patches:
 - path: ../../default/webhookcainjection_patch.yaml
-namespace: services
```

Comment:
We should probably keep this in, as a mechanism to autoreplace namespaces in the manifest we generate.

Reply:
good idea! I applied the same suggestion to Application Agent's manifests